### PR TITLE
Fix grammar of asynchronous iterator glossary entry

### PR DESCRIPTION
**Pull request by andresdelfino**

### CODE REVIEW

**Review by Mariatta — APPROVED** [review]

**Comment by Mariatta** [comment]
I think this looks good, but I'll let @python/asyncio team have a quick look before I merge.

**Review by 1st1 — APPROVED** [review]

**Comment by 1st1** [comment]
LGTM.

**Comment by Mariatta** [comment]
Thanks!

**Pull request merged**
